### PR TITLE
Remove app insights key as it is already made dynamic in tf file

### PR DESCRIPTION
**Pull request by mounikahmcts**
Reference:
https://github.com/hmcts/fpl-ccd-configuration/pull/990

### CODE REVIEW

**Comment by Gogetter** [comment]
@mounikahmcts do we need to add this to key vault and reference in `values.yaml`?

**Comment by mounikahmcts** [comment]
> @mounikahmcts do we need to add this to key vault and reference in `values.yaml`?

@Gogetter  - I have already done that as part of https://github.com/hmcts/fpl-ccd-configuration/pull/990/files

File: k8s/ithc/common/family-public-law/fpl-case-service.yaml [hunk 1]
```diff
@@ -14,15 +14,14 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: fpl-case-service
-    version: 1.8.0
+    version: 1.10.4
```
Comment (Gogetter) [inline k8s/ithc/common/family-public-law/fpl-case-service.yaml:17]: Looks like we are currently on [1.10.5](https://github.com/hmcts/fpl-ccd-configuration/blob/master/charts/fpl-case-service/Chart.yaml)

Comment (mounikahmcts) [reply k8s/ithc/common/family-public-law/fpl-case-service.yaml]: updated it

**Pull request merged**